nginx.conf

```diff
@@ -34,6 +34,8 @@ server {
   rewrite ^/server/(.*)$ https://github.com/getsentry/onpremise/ redirect;
   # /self-hosting redirects
 
+  rewrite ^/ip-ranges/?$ /meta/ip-ranges/$is_args$args redirect;
```

Comment:
of note I threw this under meta as 'security-and-legal' is awfully chunky

Reply:
Agreed. 

Of note: GitHub provides the DPA as part of the organization settings information - we might consider that for future. They also create a step-by-step article on how to accept it: https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/entering-a-data-protection-agreement-with-github-for-gdpr-compliance

They also require that the the Terms of Service have been accepted first; do we?

Reply:
this is similar to what we do except that we are pure digital